pages/common/keepass2.md

```diff
@@ -0,0 +1,15 @@
+# keepass2
+
+> A light-weight password manager.
+
+- Start KeePass 2, opening the most recently-opened password database:
+
+`keepass2`
+
+- Start KeePass 2, opening a specific password database:
+
+`keepass2 "{{/path/to/database.kbdx}}"`
+
+- Use a specific key file to open a password database:
+
+`keepass2 "{{/path/to/database.kbdx}}" -keyfile:{{/path/to/key/file.key}}`
```

Comment:
> Shouldn't there be quotes around the key file path too ?

Nope

> And how is the key file generated for a particular db ?

It's created when you first set up a database,  I think. I don't use it personally.

> Or neither. We don't usually quote paths. Is it particularly necessary in this case?

Nope! This is just my personal preference.

> By the way, do the paths need to be absolute? If not, maybe remove the leading slash.

No again!